frontend/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Running `npm test`, this fails. Can you update the test so it is actually testing what you've written, rather than the default react test here.

Reply:
I've never written tests for frontend before, but that test now passes.